### DISCUSSION TITLE
Department not displayed.

### DISCUSSION THREAD

**Discussion by adumith** [microsoft-search/pnp-modern-search#3882] [Q&A]
Hi everyone,

Even though I’ve selected the “department” field, it’s not showing up in the list results. It’s only showing the person's name; I want it to show both the name and the department.

I’ve even used the SharePoint query tool, and the “Department” property is populated, but for some reason, it’s not showing in the results.

Any idea why?

Thanks a lot.

**Comment by adumith** [comment]
Additional Information:

I have configured the results to be sorted by department, and the search is performing this sorting without any problems. The only issue is that the department field is not being displayed.

Do you have any idea where I can look to find the cause of this?

**Comment by adumith** [comment]
I did it by my self

**Reply by kasperbolarsen** [reply]
What was the issue? Others might hit the same issue so we would like to know the cause

**Reply by adumith** [reply]
I didn't add the field into the layout slots.
